lib/active_merchant/billing/gateways/redsys_rest.rb

```diff
@@ -82,7 +82,8 @@ class RedsysRestGateway < Gateway
         authorize:  '1',
         capture:    '2',
         refund:     '3',
-        cancel:     '9'
+        cancel:     '9',
+        verify:     '7'
```

Comment:
🤔  I was torn on whether to make `verify` a transaction type of 7 or not, what led you to use 7?

Reply:
Because in [their docs](https://pagosonline.redsys.es/funcionalidades-preautenticacion.html) it points out that for Rest it should be transaction type of 7.

Reply:
When I read that I interpreted it as like a pre-authorization because you can pass in an amount and then you can "complete" the charge by doing a follow up with transaction 8. We can go with a `7` type though!